platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -231,12 +233,17 @@ private MigrationReportImpl doExport(
     }
     report.end();
     if (report.hasErrors()) {
+      SecurityLogger.audit("Errors exporting configuration settings to file {}", exportFile);
```

Comment:
The specific errors are audited separately at the time. This one is just to audit the whole import process failed

Reply:
I get that. What I'm saying is that only the audits for the specific errors are needed, in my opinion. i.e. I don't think it's necessary to have the last line in this sequence of audits:
```
Called Import Command.
An error occurred with file1
An error occurred with file2
Errors importing configuration settings
```
That last line is not adding any value.
@stustison what do you think?

Reply:
Your scenario is not necessarily the other one. There might be failures during import or export that are not file related and these are not audited. In such case, there wouldn't be any audit about the export/import failure. 

Since I cannot make the distinction of why it fails, I would rather simply audit the fact that the whole export/import failed.

Reply:
I'm OK with this. Anytime we write to disk we should provide justification.